src/utils/array.js

```diff
@@ -63,6 +63,19 @@ export function findIndex(array, predicate) {
   return -1;
 }
 
+/**
+ * Returns the first element that matches the predicate, or undefined if no
+ * element matches the predicate. Like Array#find.
+ *
+ * @param {!Array<T>} array
+ * @param {function(T, number, !Array<T>): boolean} predicate
+ * @return {T|undefined}
+ * @template T
+ */
+export function find(array, predicate) {
+  return array[findIndex(array, predicate)];
```

Comment:
I imagine the `find` function doesn't work well with an array that contains undefined. [true, undefined, true] for example. This is a one line function, and I don't feel the need to make it a helper function.